docs/writing-tests/accessibility-testing.md

```diff
@@ -148,37 +156,65 @@ Disable accessibility testing for stories or components by adding the following
+### Setup
 
-Accessibility testing with Storybook shortens the feedback loop which means you fix issues faster. Reuse stories in a Jest test and run an accessibility audit on them using the [jest-axe integration](https://github.com/nickcolley/jest-axe). That also unlocks the ability to integrate accessibility tests into your functional testing pipeline.
+To enable accessibility testing with the test runner, you will need to take additional steps to set it up properly. Detailed below is our recommendation to configure and execute them.
 
-For example, include the following test file to run an accessibility test on a story:
+Run the following command to install the required dependencies.
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
-    'react/accessibility-testing-with-jest-axe.js.mdx',
-    'vue/accessibility-testing-with-jest-axe.js.mdx',
+    'common/storybook-test-runner-axe-playwright.yarn.js.mdx',
+    'common/storybook-test-runner-axe-playwright.npm.js.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-When you execute your test script, it will run the accessibility audit along with any interaction tests you might have.
+Add a new [configuration file](./test-runner.md#test-hook-api-experimental) inside your Storybook directory with the following inside:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/storybook-test-runner-a11y-config.js.mdx',
+    'common/storybook-test-runner-a11y-config.ts.mdx',
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+<div class="aside">
+
+💡 `preRender` and `postRender` are convenient hooks that allow you to extend the test runner's default configuration. They are **experimental** and subject to changes. Read more about them [here](./test-runner.md#test-hook-api-experimental).
+
+</div>
+
+When you execute the test runner (for example, with `yarn test-storybook`), it will run the accessibility audit and any [interaction tests](./interaction-testing.md) you might have configured for each component story.
+
+It starts checking for issues by traversing the DOM tree starting from the story's root element and generates a detailed report based on the issues it encountered.
 
-![Accessibility testing with Jest Axe Core](./jest-accessibility-testing-optimized.png)
+![Accessibility testing with the test runner](./test-runner-a11y-optimized.png)
 
 ---
 
 #### What’s the difference between browser-based and linter-based accessibility tests?
 
-Browser-based accessibility tests, like found in Storybook, evaluates the rendered DOM because that gives you the highest accuracy. Auditing code that hasn't been compiled yet is one step removed from the real thing so you won't catch everything the user might experience.
+Browser-based accessibility tests, like found in Storybook, evaluate the rendered DOM because that gives you the highest accuracy. Auditing code that hasn't been compiled yet is one step removed from the real thing, so you won't catch everything the user might experience.
```

Comment:
```suggestion
Browser-based accessibility tests, like those found in Storybook, evaluate the rendered DOM because that gives you the highest accuracy. Auditing code that hasn't been compiled yet is one step removed from the real thing, so you won't catch everything the user might experience.
```